src/main/java/calculator/model/Operator.java

```diff
@@ -0,0 +1,42 @@
+package calculator.model;
+
+import java.util.Arrays;
+import java.util.function.BiFunction;
+
+public enum Operator {
+    PLUS("+", (number1, number2) -> new Number(number1.getNumber() + number2.getNumber())),
+    MINUS("-", (number1, number2) -> new Number(number1.getNumber() - number2.getNumber())),
+    DIVIDE("/", (number1, number2) -> {
+        try {
+            return new Number(number1.getNumber() / number2.getNumber());
+        } catch (ArithmeticException arithmeticException) {
+            return new Number(0);
+        }
+    }),
+    MULTIPLE("*", (number1, number2) -> {
+        try {
+            return new Number(number1.getNumber() * number2.getNumber());
+        } catch (ArithmeticException arithmeticException) {
+            return new Number(0);
+        }
+    });
+
+    private final String operator;
+    private final BiFunction<Number, Number, Number> function;
+
+    Operator(String operator, BiFunction<Number, Number, Number> function) {
+        this.operator = operator;
+        this.function = function;
+    }
+
+    public static Operator create(String operator) {
```

Comment:
파라미터 단계에서는 아직 기호라는 의미가 있을 것 같습니다. 😄 
```suggestion
    public static Operator create(String sign) {
```